### PR TITLE
Add search functionality

### PR DESCRIPTION
**Pull request by numist**
Looks like you're getting a lot of requests to add new characters! Here's a basic search box (with basic sorting by relevance) lifted from https://github.com/numist/numi.st/pull/85 in case scrolling stops being good enough.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,38 +93,148 @@
...
+      background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-search" viewBox="0 0 16 16"><path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/></svg>');
+      background-repeat: no-repeat;
+      background-position: 16px center;
+      background-size: 16px 16px;
+    }
   </style>
 
   <script>
-    const symbols = [
-      "©", "®", "™", "“", "”", "£", "…", "½", "¼", "∞", "é", "á", "à", "ç", "€",
```
Comment (numist) [inline index.html:100]: Moving toward one line per character might also help reduce merge conflicts in the future (or at least make them easier to sort out) 🫠

**Comment by qwell** [comment]
I think #29 supersedes this quite nicely.  That one is a bit more expansive and flexible.

**Comment by samwho** [comment]
I went ahead and merged #29. I greatly appreciate the time you spent on this PR @numist 🙏

**Comment by samwho** [comment]
@numist any improvements to search are most welcome. Sorry, I didn't realise you'd done fuzzy searching here. Feel free to open a new PR!

**Comment by numist** [comment]
Opened https://github.com/samwho/symbol.wtf/pull/62.

**Pull request closed without merging**